explorer/client/src/utils/rpc_basictest.ts

```diff
@@ -0,0 +1,12 @@
+import { SuiRpcClient } from "./rpc";
```

Comment:
Should we remove this or write better test?

Reply:
remove, i was merely using this to get the rpc lib up and running

Reply:
removed: https://github.com/MystenLabs/sui/commit/a8e02eaf8e18e9cf7e6b93d5f541bbde5210ab9f